index.html

```diff
@@ -25,29 +25,25 @@
         </nav>
     </header>
    <!--Heading-intro-->
-<section>
-<div id="Intro">
-    <div id="introtext">
-        <div id="introparagraph">
-            <h1>Hey There.
-              I'm Andi</h1>
-              <br>
-                <h2>I'am a Software Developer</h2>
+<section class="introduction">
+<div class="intro">
+    <div class="introtext">
+        <div class="introparagraph">
+            <h1 class="headsec"> Hey There.
+                I'm Andi</h1>
+                <br>
+                <h3 class="headthree">I'am a Software Developer</h3>
                 <br>
-                <p><h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don't hestiate to contact me.</h3></p>
+                <p id="onepara">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.</p>
         </div>
     </div>
-    <div id="background-image">
-        <div class="picture">
-            <img src="">
-        </div>
     </div>
 </div>
 </section>
 <br>
 <div class="social-media">
-    <ul>
+    <ul class="social">
         <li>
             <a href="https://github.com/AndisiweJ">
                 <i class="fa-brands fa-github"></i>
```

Comment:
- [ ] **[OPTIONAL]** There are five social icons in the Figma template: Github, LinkedIn, Angellist, Tweeter, and Medium, please 🙏🏽 add all of them to your design to look similar to the Figma design.

| **Figma design** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192119280-de86b4a3-c0f0-4f97-9eb5-865d986c16f9.png) | ![image](https://user-images.githubusercontent.com/58642949/192119296-4de1c959-39b6-4875-b77e-d952f4afa5b4.png) |